tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/CI/TestingFrameworkImpactedTests.cs

```diff
@@ -45,6 +45,34 @@ public TestingFrameworkImpactedTests(string sampleAppName, ITestOutputHelper out
             SetEnvironmentVariable(ConfigurationKeys.CIVisibility.Logs, "1");
         }
 
+        internal ProcessHelpers.CommandOutput RunGitCommand(string arguments)
+        {
+            try
+            {
+                var gitOutput = ProcessHelpers.RunCommand(
+                    new ProcessHelpers.Command(
+                        "git",
+                        arguments,
+                        EnvironmentTools.GetSolutionDirectory(),
+                        outputEncoding: Encoding.Default,
+                        errorEncoding: Encoding.Default,
+                        inputEncoding: Encoding.Default,
+                        useWhereIsIfFileNotFound: true),
+                    null);
+
+                if (gitOutput is null || (gitOutput.ExitCode < 0 && gitOutput.Error is not { Length: > 0 }))
```

Comment:
```suggestion
                if (gitOutput is null || (gitOutput.ExitCode != 0 && gitOutput.Error is not { Length: > 0 }))
```

shouldn't be `!=` here?

Reply:
Fair, changing it!